src/map/lapack2flash/FLASH_bdsqr.c

```diff
@@ -0,0 +1,199 @@
+/*
+
+    Copyright (C) 2014, The University of Texas at Austin
```

Comment:
add 2022 AMD Copyright (and in files below) for the ones that have already been FLASH-ified

Reply:
Fixed: https://github.com/iotamudelta/libflame/pull/2/commits/d97dcf51146faa3606e9d7194e483c2c433cf845

Reply:
Added for the file files I converted to FLASH